data/Rumman_2018/metadata.yml

```diff
@@ -0,0 +1,290 @@
+    in functional trait values with climate.
+  basis_of_record: field
+  life_stage: adult
+  sampling_strategy: The study accessed eight nodes of the Australian SuperSite Network
+    across the continent. This Network (http://www.tern-supersites.net.au/) is a part
+    of the Terrestrial Ecosystem Research Network (TERN), a federally funded initiative
+    of the Australian government. Each SuperSite is a distinctly different type of
+    ecosystem. Multiple data streams collected from these SuperSites provide a novel
+    opportunity to combine and compare contrasting ecosystems across continental Australia.
+    ^p Each site is comprised of a one hectare plot within the footprint of the eddy
+    covariance flux tower located at each SuperSite. Within each hectare, replicated
+    measurements were made on dominant and mostly broadleaf evergreen overstorey species
+    that collectively accounted for ca 80% of the (aboveground) biomass of each site.
+    Across all sites, a total of 55 plant species were sampled in the dryseason and
+    44 species sampled in the wet-season (417 individual trees from 27 plant families).
+    Identical measurements were made of foliar d13C on four to six selected individuals
+    of each species, from which D13C was calculated. Variation in 16 additional leaf
+    attributes (gas-exchange characteristics, structural and nutrient traits, see
+    Table 2 for details) across the SuperSites and their relationships with D13C were
+    also examined. ^p Leaf gas-exchange and additional leaf traits (Table 2) were
+    measured on 407 individual plants across 80 species at each of the seven Supersites
+    (Table 1), generally across winter and summer seasons or wet- and dry-seasons.
+    Leaf gas-exchange was measured in the morning using a portable photosynthesis
+    system (Li-Cor 6400; Li-Cor, Nebraska, USA) on upper canopy, fully expanded leaves
+    on detached branches. Where there was evidence of an impact of detachment on gas
+    exchange, replacement branches were sampled. ^p Mature, healthy, upper canopy,
+    sunlit leaves were collected in the wet/dry or summer/winter seasons for analysis
+    of d13C isotope composition.
+  original_file: Transcribed from manuscript and supplementary material; located in
+    raw folder and archived on Google Drive.
+  notes: The analyses for this study include many additional traits. Most of this data is in Bloomfield_2018. The only data analyses in this study, but not available is from Litchfield Savanna. Although we haven't made contact with the author, Owen Atkin and Keith Bloomfield confirmed the source of the other data and suggested Derek Eamus would have the additional data from Litchfield Savanna. 
+locations:
+  Alice Mulga:
+    site code: AMU
+    longitude (deg): 133.2493
+    latitude (deg): -22.2828
+    precipitation, MAP (mm): 321.0
+    temperature, MA (C): 22.4
+    description: semi-arid tropical savanna
```

Comment:
Daniel and I decided early on to use "description" for a really brief vegetation description - since this is 1 of 3 fields that is "highlighted" in the reports (others are lat/lon). It often feels a little weird, but we haven't wanted to do mass changes throughout. (You might have noticed I've renamed some in the datasets you added to match this convention.)